helmfile/50-applications.yaml

```diff
@@ -454,7 +454,7 @@ releases:
     component: backup
   chart: ./charts/harbor-backup
   version: 0.1.0
-  installed: {{ and .Values.harbor.enabled .Values.harbor.backup.enabled }}
+  installed: {{ and .Values.harbor.enabled (and .Values.harbor.backup.enabled (eq .Values.harbor.database.type "internal")) }}
```

Comment:
Isn't it a little confusing that if `harbor.backup.enabled` is enabled and the database type is set to external that it doesn't take a backup? In that case I would like to have an error telling be that this is not a valid configuration.

Reply:
Yes that is a good point. I will add a warning for it.

Reply:
Added the fail warning now